assembly_uploader/ena_queries.py

```diff
@@ -86,36 +54,136 @@ def parse_accession(accession):
 
 
 class EnaQuery:
-    def __init__(self, accession, username=None, password=None):
-        self.url = ENA_API_URL
+    def __init__(self, accession, private=False):
+        self.private_url = "https://www.ebi.ac.uk/ena/submit/report/"
+        self.public_url = "https://www.ebi.ac.uk/ena/portal/api/search"
```

Comment:
I guess at some point we should put these and the max retry count into a config.py using pydantic-settings, so they can be overridden (e.g. to use wwwdev instances). One for the future just for consistency with some other recent projects – no need to do it now.